msteams-platform/TOC.yml

```diff
@@ -266,6 +266,9 @@
             href: toolkit/toolkit-v4/provision-vs.md
           - name: Deploy to the cloud
             href: toolkit/toolkit-v4/deploy-vs.md            
+    - name: Migrate to new project structure
```

Comment:
The migration guide should be places under TTK for VS folder.
and suggest the page title to "Migrate project from 17.9 to 17.10+"

Reply:
Updated